CONTRIBUTING.md

```diff
@@ -1,7 +1,28 @@
-# Contributing
+# Contributing to panther-analysis
 
-Please follow the [Code of Conduct](https://github.com/panther-labs/panther-analysis/blob/main/CODE_OF_CONDUCT.md)
-in all of your interactions with the project.
+Thank you for your interest in contributing to Panther's open-source ruleset!  We value and appreciate all contributions, whether they are bug reports, feature requests, or detection rule contributions.
```

Comment:
aww this is nice. sugg update to second sentence: "We appreciate all types of contributions, including new detection rules, feature requests, and bug reports."